website/docs/r/virtual_network_gateway_connection.html.markdown

```diff
@@ -92,7 +92,7 @@ in different locations/regions.
 ```hcl
 resource "azurerm_resource_group" "us" {
   name     = "us"
-  location = "East US"
+  location = "West Europe"
```

Comment:
These kind of examples will be tricky to understand...

Reply:
Namely, this demonstrates peering/gateways between regions and now they are both referring to the same region.

Reply:
looks like this has been resolved